src/main/java/org/mule/service/http/impl/service/client/GrizzlyHttpClient.java

```diff
@@ -475,7 +478,7 @@ private void handleRedirectAsync(HttpRequest request, HttpResponse response, Htt
   protected Request createGrizzlyRequest(HttpRequest request, HttpRequestOptions options)
       throws IOException {
     RequestBuilder reqBuilder = createRequestBuilder(request, options, builder -> {
-      builder.setFollowRedirects(false);
+      builder.setFollowRedirects(!enableMuleRedirect && options.isFollowsRedirect());
```

Comment:
Does this mean that now, by default we are delegating the redirect logic to grizzly as before MULE-19622? 
Explain this in a comment for the system property.

Reply:
The answer is yes as indicated in the comment for the system property.